src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -1215,30 +1215,42 @@ private function verifyTypes(string $type, mixed $value, ?array &$invalidTypes =
      */
     private function splitOutsideParenthesis(string $type): array
     {
+        // Fast path for strings without pipes (most common case)
```

Comment:
I think the next line in self-explanatory?

Reply:
ok